coordinateutils/include/coordinateutils/chealpix.h

```diff
@@ -64,13 +64,16 @@ typedef struct {
   long nring;
   double fact1;
   double fact2;
+  int nest;
+  int shift_phi;
```

Comment:
Aren't these actually bools?

Reply:
That requires enabling C99 in the `cmake` configuration.